src/compiler/checker.ts

```diff
@@ -1070,7 +1069,7 @@ import {
     WhileStatement,
     WideningContext,
     WithStatement,
-    YieldExpression,
+    YieldExpression

```

Comment:
Do you know what you ran which did this? Was it organize imports? Something else? Hoping to get to the bottom of these so we can be consistent (or, I guess formatting will prevent that).

Reply:
Probably organize imports.

Reply:
Hm, just tested running organize on `checker.ts` and this didn't happen, even when reordering things, really odd.

Reply:
I think it was a result of running Organize Imports to remove an unused import. I just checked locally and that does remove the trailing `,`.